### ISSUE TITLE
运行了代码但没有检测结果

### ISSUE THREAD

**Issue by WangXiaoLongNWPU** [DetectionTeamUCAS/RetinaNet_Tensorflow_Rotation#16]
安装并运行了代码，测试了几张无人机俯视拍摄的包含汽车的图片和dota数据集中的几张图片，使用--show_box成功保存处理后的图像，但是图上都没有画出框。检查了test_dota.py中间变量，发现det_boxes_r_均返回空数组。我的环境是anaconda3+ python3.5+ tensorflow1.12+cuda 9.1。cfgs.py文件中只修改了NET_NAME = 'resnet_v1_50'。换成其他预训练网结果相同：程序可以跑通但是没有检测到任何结果。

同样的图片在R2CNN_Faster-RCNN_Tensorflow算法上运行有结果，图上有画框。

请问有什么建议吗？

**Comment by yangxue0827** [comment]
你用的我更新后的代码吗？ @WangXiaoLongNWPU

**Comment by WangXiaoLongNWPU** [comment]
我两天前下载过，刚刚重新下载运行，结果是一样的。

**Comment by yangxue0827** [comment]
我说的是训练代码你用的最新的吗 @WangXiaoLongNWPU

**Comment by WangXiaoLongNWPU** [comment]
没有训练，直接测试。这样不行吗？是必须首先训练吗？

**Comment by yangxue0827** [comment]
之前的训练代码有点问题，后来更新了，我看另一个人说效果可以 @WangXiaoLongNWPU

**Comment by yangxue0827** [comment]
https://github.com/DetectionTeamUCAS/RetinaNet_Tensorflow_Rotation/issues/8

**Comment by WangXiaoLongNWPU** [comment]
我可以试着训练一下。不过这个代码正常情况是不训练也可以直接测试的吧？

**Comment by yangxue0827** [comment]
你没训练好的权重怎么测试？ @WangXiaoLongNWPU

**Comment by WangXiaoLongNWPU** [comment]
明白了！谢谢

**Comment by clw5180** [comment]
更新了反而nan了，更新之前还没出现nan过

**Comment by yangxue0827** [comment]
https://github.com/DetectionTeamUCAS/RetinaNet_Tensorflow_Rotation/issues/8
问一下他的结果？ @clw5180

**Comment by clw5180** [comment]
> #8
> 问一下他的结果？ @clw5180

更新了代码，经常会出现个别点reg loss = nan的情况。更新之前训练了500k steps没出现过nan

**Comment by xincyu** [comment]
> 你没训练好的权重怎么测试？ @WangXiaoLongNWPU

利用ReadMe里百度网盘提供的预训练权，难道也不能直接测试DOTA吗？那个权不是DOTA的吗
